tests/NuGet.Services.AzureSearch.Tests/SearchService/SearchTextBuilderFacts.cs

```diff
@@ -232,7 +232,10 @@ public static IEnumerable<object[]> CommonAzureSearchQueryData()
                     { "title:foo bar", "+title:foo bar" },
                     { "title:foo unknown:bar", "title:foo" },
 
-                    // If there are non-field-scoped terms and no field-scoped terms, at least of one the non-field-scoped terms is required.
+                    // If there is a single unscoped term, favor results that match all token fragments after splitting on periods.
+                    { "Foo.Bar", "Foo.Bar (+Foo +Bar)^2" },
```

Comment:
Tests for leading dots `.Foo`, trailing dots `Foo.`, double dots `Foo..Bar`, just dots `..`, quotes around `"Foo.Bar"`

Reply:
Looks like we need the "dots but multiple terms" case `Foo.Bar Baz`. Maybe i missed it

Reply:
Good catch, I've added more tests!